core/src/main/java/org/apache/iceberg/PartitionsTable.java

```diff
@@ -73,6 +73,11 @@ public class PartitionsTable extends BaseMetadataTable {
                 "equality_delete_file_count",
                 Types.IntegerType.get(),
                 "Count of equality delete files"),
+            Types.NestedField.required(
+                11,
+                "total_data_file_size_in_bytes",
+                Types.LongType.get(),
+                "Total size in bytes"),
```

Comment:
Ah sorry, in my previous comment I meant just change "total bytes' => 'total size in bytes', but the rest was ok.

So can we revert back the original end of sentence?  

'Total size in bytes of data files'  (maybe 'in a partition' was redundant there)